Makefile

```diff
@@ -71,7 +64,6 @@ format: format-ruff format-codespell  ## Run all automatic formatters
 
 .PHONY: format-ruff
 format-ruff:  ##- Automatically format with ruff
```

Comment:
Should the `format-...` targets also depends on `install-...` ones? (`install-ruff` in the case of `format-ruff`)